src/components/footnote-link/footnote-link.scss

```diff
@@ -0,0 +1,61 @@
+@use '../../compiled/tokens/scss/color';
+@use '../../compiled/tokens/scss/ease';
+@use "../../compiled/tokens/scss/font-weight";
+@use '../../compiled/tokens/scss/size';
+@use "../../compiled/tokens/scss/scale";
+@use '../../compiled/tokens/scss/transition';
+@use '../../mixins/focus';
+@use '../../mixins/ms';
+@use '../../mixins/theme';
+
+/// Footnote links are small numbered circles within a body of text that
+/// link to a footnote below
+///
+/// 1. They start off as circles but if their content gets wide they turn into pills
+/// 2. Since they can be linked to we assign them a margin offset, so when they're
+///    linked they're not right at the top of the screen
+/// 3. They have a higher vertical baseline than the text around them. (This is
+///    similar to the `<sup>` element. Since `<sup>` has no semantic meaning, we
+///    apply these styles via CSS
+.c-footnote-link {
```

Comment:
@Paul-Hebert Might it be possible and make sense to visually inform which footnote link I am sent back to after clicking on a footnote return link?

I was expecting something like the `:focus` styles on the footnote link after I clicked the footnote return link. What are your thoughts? 🤔 

![which](https://user-images.githubusercontent.com/459757/131891534-247b8e04-f137-45fc-ab4d-c03b0993a0d4.gif)

Reply:
Hmm, good question...the link is not actually focused so I don't want to send mixed messages there... I wonder if there's another way we could indicate that. I've got an idea. Let me try something...

Reply:
@gerardo-rodriguez , @tylersticka , what if we used an animation when a footnote link becomes targeted. This way we could initially represent the connection, but then quickly remove the styles to avoid confusion between the `:focus` and `:target` styles. A couple ideas:

![targeted-link-2](https://user-images.githubusercontent.com/5798536/131893783-d5f35817-5867-45b0-8c35-fd32868f71fc.gif)
![targeted-link](https://user-images.githubusercontent.com/5798536/131893789-97db06d9-5c7e-4b25-85b0-3c475de5206b.gif)

Reply:
@Paul-Hebert I like that!

Reply:
Oh, I like your idea, @Paul-Hebert!

Reply:
I'm okay with either of the ideas, style-wise. They both work. 🙂

Reply:
I implemented the box-shadow animation :+1:

https://github.com/cloudfour/cloudfour.com-patterns/pull/1518/commits/bc7f19459d687ab37c6d468dbedf7a785e03ef29